apps/backoffice/src/app/api/auth/[...nextauth]/route.ts

```diff
@@ -31,37 +32,55 @@ const authOptions: NextAuthOptions = {
       id: "access-control",
       credentials: {},
       authorize(credentials) {
-        const { session_token } = credentials as {
-          session_token: string;
-        };
+        const {
+          identity_token: identity_token
+        } = credentials as SelfCareIdentity;
 
         // first quick token check
-        if (isNullUndefinedOrEmpty(session_token)) {
-          throw new Error("null undefined or empty session token");
+        if (isNullUndefinedOrEmpty(identity_token)) {
```

Comment:
why not `NonEmptyString.is` ?

Reply:
08b247d16c55a3f2434bb556d5c8508f8fbddbf1